frontend/components/cards/ModuleCard.tsx

```diff
@@ -0,0 +1,180 @@
+import { BaseModule } from "@/interfaces";
+import { Card } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { FC, ReactNode } from "react";
+
+const useStyles = makeStyles({
```

Comment:
Created [`MH-212`](https://modularhistory.atlassian.net/browse/MH-212) for moving this styling to a CSS module.